torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -852,6 +854,91 @@ def call_function(
         )
 
 
+class SdpaHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def create_wrapped_node(self, tx, args, kwargs):
+        # See NOTE [HigherOrderOperator tracing design] for more details
+        checkpoint = tx.copy_graphstate()
+        graph_checkpoint = tx.output.graph
+
+        example_vals = [torch.zeros((), dtype=args[0].dtype)] + [
+            torch.zeros((), dtype=torch.int) for _ in range(4)
+        ]
+        with tx.output.new_subtracer() as tracer:
+
+            def create_dynamo_inp(example_value, name):
```

Comment:
Does this actually do anything? It looks like we're creating a tracer and throwing it away (speculate_subgraph creates its own)